### ISSUE TITLE
Testing with KITTI point cloud files, the results of orientation is not aligned with bounding box

### ISSUE THREAD

**Issue by chowkamlee81** [tianweiy/CenterPoint#4]
I started visualizng results of PointPillars "nusc_centerpoint_pp_02voxel_circle_nms.py" on KITTI/ VLP 32 our own recorded bags. Results are not good.
Is algorithm has generalization capability to work with other pointcloud opensource dat like KITTI ROS bags(VLP64)/VLP32 data.
Algorithm does well on Nuscenes rosbag files/nuscene pointcloud data. 
But from the results on other data looks it is not generalizing
Kindly help.

**Comment by tianweiy** [comment]
Yes, I also noticed large domain gap between Datasets. Here is one paper from cvpr that you can check out. https://arxiv.org/abs/2005.08139#:~:text=In%20the%20domain%20of%20autonomous,and%20stereo%20camera%20data%20alike.

**Comment by tianweiy** [comment]
```the results of orientation is not aligned with bounding box``` <- nuScenes and KITTI uses quite different coordinate systems, you may need to check the transformation.

**Comment by chowkamlee81** [comment]
Will look into the suggestions and update you

**Comment by tianweiy** [comment]
Great, that you get it works!

**Comment by YoushaaMurhij** [comment]
@chowkamlee81 Could you please share how to test on KITTI dataset and how to prepare data?

**Comment by tianweiy** [comment]
@chowkamlee81  I add some preliminary Waymo support today you can see if they work better

**Comment by chowkamlee81** [comment]
@tianweiy ,
Your codes are really handy at right time. Will update you on the result
